Client/src/Components/product-reviews/ReviewList.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import Review from "./review";
+
+// Creates a list of reviews. Maps in the list from ProductReviews to Review to create each review
+function ReviewList({ list }) {
+  const [listCounter, setList] = useState("Latest");
+  const listNewDateFormat = [...list];
+  list.forEach((review, index) => {
+    listNewDateFormat[index].date = new Date(review.date);
+  });
+  const helpfulList = [...listNewDateFormat].sort((a, b) => b.helpfulness - a.helpfulness);
+  const latestList = [...listNewDateFormat].sort((a, b) => b.date - a.date);
+
+  function onChange() {
+    const e = document.getElementById("test");
+    const selectedText = e.options[e.options.selectedIndex].text;
+    setList(selectedText);
+  }
+
+  return (
+    <div>
+      <select id="test" onChange={onChange}>
+        <option value="latest">Latest</option>
+        <option value="most-helpful">Most Helpful</option>
+        <option value="recommended">recommended</option>
+      </select>
+      <div>
+        {listCounter === "Latest" && (latestList.map((review) => (
+          <div>
```

Comment:
Trying using React.Fragment instead of divs that have no other function other than to encapsulate jsx element.

Reply:
I switched to React.Fragment but for some reason they prefer if I use <> </> instead. So I switched it to that instead.
For example: 
<>
    <Review review={review} />
    <hr />
</>